src/hyperfine/benchmark.rs

```diff
@@ -370,15 +387,15 @@ pub fn run_benchmark(
         warnings.push(Warnings::OutliersDetected);
     }
 
-    if !warnings.is_empty() {
+    if !warnings.is_empty() && options.output_style != OutputStyleOption::None {
```

Comment:
I think we should always print the warnings. They are written to STDERR anyway.

Reply:
Agree on that point, changed in code